tests/component_handlers/kafka_connect/test_connect_wrapper.py

```diff
@@ -48,7 +47,7 @@ def test_should_through_exception_when_host_is_not_set(self):
             == "The Kafka Connect host is not set. Please set the host in the config."
         )
 
-    @patch("requests.post")
+    @patch("httpx.post")
```

Comment:
How about we use `@pytest.fixture` for this?

Reply:
the thing is you would need to know what operation to patch. You need to say if you want a post or get (etc) request. I would need to write it manually if I do like that and declare the async mock manually

Reply:
This is similar to the logs (like log_info, log_debug, etc.), but it is not essential to change. I just wanted to keep it consistent like the other tests.